torchbenchmark/__init__.py

```diff
@@ -136,6 +136,28 @@ def name(self) -> str:
         return os.path.basename(self.path)
 
 
+class Worker(subprocess_worker.SubprocessWorker):
+    """Run subprocess using taskset if CPU affinity is set.
+
+    When GOMP_CPU_AFFINITY is set, importing `torch` in the main process has
+    the very surprising effect of changing the threading behavior in the
+    subprocess. (See https://github.com/pytorch/pytorch/issues/49971 for
```

Comment:
Maybe it is also worth mentioning using "torch.set_num_threads()" won't saturate the CPU resources?

Reply:
Oh, that's a great point!